app/recordtransfer/management/commands/send_test_email.py

```diff
@@ -0,0 +1,156 @@
+)
+
+from recordtransfer.models import InProgressSubmission, Submission, UploadSession, User
+
+
+logger = logging.getLogger(__name__)
+
+EMAIL_FUNCTIONS = {
+    "submission_creation_success": send_submission_creation_success,
+    "submission_creation_failure": send_submission_creation_failure,
+    "thank_you_for_your_submission": send_thank_you_for_your_submission,
+    "your_submission_did_not_go_through": send_your_submission_did_not_go_through,
+    "user_activation_email": send_user_activation_email,
+    "user_account_updated": send_user_account_updated,
+    "in_progress_submission_expiring": send_user_in_progress_submission_expiring,
+}
+
+
+class Command(BaseCommand):
+    help = "Send a test email to the specified address using the given email ID."
+
+    def add_arguments(self, parser: argparse.ArgumentParser) -> None:
+        """Add command-line arguments to the parser."""
+        parser.add_argument(
+            "to_email", type=str, help="The email address to send the test email to."
+        )
+        parser.add_argument(
+            "email_id", type=str, help="The ID of the email to send (email function name)."
+        )
+
+    def handle(self, *args, **options) -> None:
+        """Handle the command to send a test email.
+
+        Args:
+            *args: Variable length argument list.
+            **options: Arbitrary keyword arguments containing 'to_email' and 'email_id'.
+        """
+        to_email = options["to_email"]
+        email_id = options["email_id"]
+
+        if email_id not in EMAIL_FUNCTIONS:
+            raise CommandError(
+                f"Unknown email_id '{email_id}'. Valid options: {', '.join(EMAIL_FUNCTIONS.keys())}"
+            )
+
+        user = self.get_test_user(to_email)
+        form_data = {"dummy": "data"}
+        submission = self.create_test_submission(user)
+        in_progress = self.create_test_in_progress_submission(user)
+
+        try:
+            self.send_email(email_id, user, form_data, submission, in_progress)
+            logger.info("✅ Sent '%s' email to %s", email_id, to_email)
+        except Exception as e:
+            logger.exception("Error sending email '%s' to %s: %s", email_id, to_email, e)
+            raise CommandError(f"Error sending email: {e}") from e
+
+    def get_test_user(self, email: str) -> User:
+        """Retrieve or create a test user with the given email address.
+
+        Args:
+            email (str): The email address of the test user.
+
+        Returns:
+            User: The retrieved or newly created User instance.
+        """
+        user, _ = User.objects.get_or_create(
+            username="testuser",
+            defaults={
+                "email": email,
+                "first_name": "Test",
+                "last_name": "User",
+                "is_active": True,
+                "gets_notification_emails": True,
+                "gets_submission_email_updates": True,
+            },
+        )
+        return user
```

Comment:
Ideally this task will have no side effects such as creating new data in the database.

Django lets you create database models *without* saving them to the database which matches exactly what we need in this command.

Here's how I'd suggest changing this function:

```suggestion
        Returns:
            User: A User instance that has not been saved to the database.
        """
        return User(
            username="testuser",
            email=email,
            first_name="Test",
            last_name="User",
            is_active=True,
            gets_notification_emails=True,
            gets_submission_email_updates=True,
        )
```